corehq/apps/reports/v2/reports/__init__.py

```diff
@@ -1,9 +1,6 @@
 from corehq.apps.reports.v2.exceptions import ReportNotFoundError
-from corehq.apps.reports.v2.reports import explore_case_data
 
-REPORTS = [
-    explore_case_data.ExploreCaseDataReport,
-]
+REPORTS = []
```

Comment:
is this populated elsewhere then?
why is this still needed?

Reply:
It's not needed. The whole app gets dropped in follow-up PR https://github.com/dimagi/commcare-hq/pull/36682